.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
+1. Describe what you expected to happen.
+
+2. Describe what actually happened.
+
+3. Describe what version of Pulp and what OS is affected by the issue.
+
+4. State steps to re-create the issue.
+
+5. If there is a related pulp.plan.io issue, link to it, and set the "smash test" field.
+
+6. Choose proper labels. See:[Smash Labels](https://pulp-smash.readthedocs.io/en/latest/about.html#labels)
```

Comment:
This will render as:

> See:[Smash Labels](https://pulp-smash.readthedocs.io/en/latest/about.html#labels)

I think there should be a space after the colon.

Reply:
done.